libcontainer/resourcemanager/resource_manager_linux.go

```diff
@@ -0,0 +1,34 @@
+// +build linux
```

Comment:
@hqhq Which is better in your opinion? Thank you.
(1) Just remove this line (+// +build linux) in resource_manager_linux.go.
(2) Add new file resource_manager_unsupported.go for !linux cases:
```
// +build !linux

package resourcemanager
```